bindings/python/dlite-collection-python.i

```diff
@@ -49,9 +57,17 @@ class Collection(Instance):
     of instances.
 
     Use the get_relations() method (or the convenience methods
-    get_subjects(), get_predicates() and get_predicates()) to iterate
-    over relations.  The number of relations is available via the `nrelations`
-    property.
+    get_subjects(), get_predicates() and get_objects()) to iterate
+    over relations.  The number of relations is available via the
+    `nrelations` property.
+
+    Relations are (s, p, o, d=None)-triples with an optional fourth field
+    `d`, specifying the datatype of the object.  It may have the following
```

Comment:
It is what? It is a bit unclear what you are referring to.  From the description nelow it seems you are referring to 'object', but it is a bit unclear.

Reply:
Updated documentation